prefq/server.py

```diff
@@ -185,4 +193,4 @@ def send_feedback():
 
 if __name__ == "__main__":
     before_first_request()
-    app.run(host="127.0.0.1", port=5000, debug=True)
+    app.run(host="127.0.0.1", port=5000, debug=True, threaded=False)
```

Comment:
Please add a comment that explains why we disable threading (due to the global variables I assume?)

Reply:
This is also not important enough to block the merge.